lib/pages/character/inventory.dart

```diff
@@ -90,42 +90,56 @@ class InventoryPage extends StatelessWidget {
   Widget _buildBag(BuildContext context, Bags bag, int bagIndex) {
     List<InventoryItem> inventory = bag.inventory.where((i) => i.id != -1 && i.itemInfo != null).toList();
     int usedSlots = inventory.length;
+    FontStyle fontStyle = (usedSlots == bag.size ? FontStyle.italic : FontStyle.normal);
```

Comment:
What's your reason for changing the font style of bags that are full?

Reply:
The widget doesn't currently show empty slots, so this is a quick visual change to display 'full' without having to cognitively parse the numbers on the right.